test/e2e_flags.go

```diff
@@ -52,6 +53,8 @@ func initializeEventingFlags() *EventingEnvironmentFlags {
 
 	flag.StringVar(&f.Tag, "tag", "e2e", "Provide the version tag for the test images.")
 
+	flag.StringVar(&f.Provisioner, "clusterChannelProvisioner", "in-memory-channel", "The name of the Channel's clusterChannelProvisioner. Only the in-memory-channel is installed by the tests, anything else must be installed before the tests are run.")
```

Comment:
nice!

wondering was there something wrong with the relocation of the kafka ccp? I've tested it (`ko apply`) and the controller did actually create the topic (for the channels)

Reply:
See #760 for the full explanation. It is more thorough then the following explanation.

The problem was that #699 made `ClusterChannelProvisioner` have a `status` subresource, but did not update the code to update the subresource, rather than the CCP itself. So the CCP never got the status `Ready`. `kafka` and `natss` (but not `in-memory-channel`) refuse to work if the CCP is not `Ready`. 

This was masked on my dev cluster because I had an older version of the `ClusterChannelProvisioner` CRD installed, which did not have the `status` subresource. So it got `Ready`. 

(I think this is true, but didn't test it) any cluster that predated the CRD change, and thus already had a `Ready` CCP, would likely have worked because the newer code simply never changed the CCP's `status`, which was already `Ready`.